include/native/wsi/native_wsi.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
What is this header for and why don't we just include the sdl2 stuff from wsi_platform.h as necessary?

Reply:
This header is for games/apps built against DXVK native to include.